docs/tutorials/elasticdl_local.md

```diff
@@ -23,10 +25,9 @@ we use a model predefined in model zoo directory.
 ### Install ElasticDL
 
 ```bash
+pip install elasticdl_client
 git clone https://github.com/sql-machine-learning/elasticdl.git
 cd elasticdl
-pip install -r elasticdl/requirements.txt
-python setup.py install
 ```
```

Comment:
### Install ElasticDL Client

```bash
pip install elasticdl_client
```

### Setup Kubernetes related environment

```bash
minikube start --vm-driver=hyperkit --cpus 2 --memory 6144 --disk-size=50gb --mount=true --mount-string="$DATA_PATH:/data"
git clone https://github.com/sql-machine-learning/elasticdl.git
cd elasticdl && kubectl apply -f elasticdl/manifests/elasticdl-rbac.yaml
eval $(minikube docker-env)
```

Mount the host path $DATA_PATH to /data of minikube

### Prepare dataset

```
docker pull elasticdl/elasticdl:dev
docker run --rm -it \
  -v $HOME/.keras/datasets:/root/.keras/datasets \
  -v $PWD:/work \
  -w /work elasticdl/elasticdl:dev \
  bash -c "scripts/gen_dataset.sh $DATA_PATH"

```


### Submit job

--volume="/data,mount_path=/data"

mount /data of minikube to /data of master pod

Reply:
Please refer to https://stackoverflow.com/questions/48534980/mount-local-directory-into-pod-in-minikube/48535001

Reply:
Great comments. updated.